### PR TITLE
Post release updates

### PR DESCRIPTION
**Pull request by cysouw**
I read the book and found all kinds of small errors, which I have corrected here. Please have a look and when acceptable, let's make this version 2.0.2 :-).

Should we also push an update to langsci?

Michael

### CODE REVIEW

File: book/chapters/implementation.tex [hunk 1]
```diff
@@ -10,11 +10,12 @@ \section{Overview}
...
 
-Alternatively for R, RStudio\footnote{\url{https://www.rstudio.com/}} provides a free and open source integrated development environment (IDE). This application can be downloaded and installed (for Mac, Windows and Unix) and it includes its own R installation and R libraries package manager. For developing in Python, we recommend the free community version of PyCharm,\footnote{\url{https://www.jetbrains.com/pycharm/}} an IDE which is available for Mac, Windows, and Unix. 
+Alternatively for R, RStudio\footnote{\url{https://www.rstudio.com/}} provides a free and open source integrated development environment (IDE). This application can be downloaded and installed (for Mac, Windows and Linux) and it includes its own R installation and R libraries package manager. For developing in Python, we recommend the free community version of PyCharm,\footnote{\url{https://www.jetbrains.com/pycharm/}} an IDE which is available for Mac, Windows, and Unix.
```
Comment (bambooforest) [inline book/chapters/implementation.tex]: @cysouw - but not Unix -> Linux at the end of the paragraph?

File: book/chapters/introduction.tex [hunk 2]
```diff
@@ -382,10 +382,10 @@ \subsubsection*{Script systems}
 
 Breaking it down further, a script consists of \textsc{graphemes}, which are writing 
 system-specific minimally distinctive symbols (see below). Graphemes may consist of one or more 
-\textsc{characters}. The term \textsc{character} is overladen. In the linguistic terminology of writing
-systems, a \textsc{character} is a general term for any self-contained element
+\textsc{characters}. The term `character' is overladen. In the linguistic terminology of writing
+systems, a character is a general term for any self-contained element
 in a writing system. A second interpretation is used as a conventional term for a unit in the Chinese writing
-system \citep{Daniels1996}. In technical terminology, a \textsc{character} 
+system \citep{Daniels1996}. In technical terminology, a character}
```
Comment (bambooforest) [inline book/chapters/introduction.tex]: @cysouw there's an opening <}> here, which stops main.tex from compiling.

File: book/chapters/pitfalls.tex [hunk 3]
```diff
@@ -137,10 +137,12 @@ \section{Pitfall: Missing glyphs}
 The Unicode Standard is often praised (and deservedly so) for solving many of
 the perennial problems with the interchange and display of the world's writing
 systems. Nevertheless, a common complaint from users is that some symbols do not display 
-correctly, i.e.\ \textit{not at all} or from a \textit{fall back font}, e.g.\ a 
+correctly, i.e. they might be displayed \textit{not at all} or only from a so-called 
+\textit{fall back font}, e.g.\ showing a 
 rectangle <▯>, question mark <?>, or the Unicode replacement character <�>. 
-The user's computer does not have the fonts 
-installed that map the desired glyphs to Unicode characters. Therefore the glyphs cannot be displayed.
+The reason for such behaviour is that the user's computer does not have the fonts
```
Comment (bambooforest) [inline book/chapters/pitfalls.tex]: @cysouw - no British English spelling ("behaviour") -- it took me a long time to get the book consistently American English spelling :)

File: book/chapters/pitfalls.tex [hunk 4]
```diff
@@ -434,7 +436,7 @@ \section{Pitfall: Canonical equivalence}
 in the first two examples by a single-character sequence; each is assigned a
 different code point. In the third example, the glyph is encoded in a
 multiple-character sequence that is composed of two character code points. All
-three sequences are \textsc{}, i.e.~they are strings that
+three sequences are \textsc{canonical equivalent}, i.e.~they are strings that
```
Comment (bambooforest) [inline book/chapters/pitfalls.tex]: @cysouw -> canonically equivalent

File: book/chapters/pitfalls.tex [hunk 5]
```diff
@@ -473,16 +475,17 @@ \section{Pitfall: Canonical equivalence}
...
+of the Unicode combining class values). Normalization forms, as very briefly
 described above, can be used to ensure character equivalence by ordering
 character sequences so that they can be faithfully compared.
 
 It is very important to note that any software application that is Unicode
 Standard compliant is free to change the character stream from one
 representation to another. This means that a software application may compose,
 decompose or reorder characters as its developers desire; as long as the
-resultant strings are  to the original. This might lead to
+resultant strings are canonical equivalent to the original. This might lead to
```
Comment (bambooforest) [inline book/chapters/pitfalls.tex]: @cysouw -> canonically equivalent

**Comment by bambooforest** [comment]
@cysouw -- looks good! a few small comments in my review. please fix.

**Comment by cysouw** [comment]
All fixed; Xelatex/Knitr compilation successful. I let you do the merge and the version change :-).

How does it work now to push these changes to the language science press?

**Comment by bambooforest** [comment]
@cysouw looks good. I also update the PDF in the main directory and then push a new version tag and a new Zenodo release. I'm not sure about LSP -- we'll have to ask Nordhoff, who is on holiday at the moment.

**Comment by bambooforest** [comment]
@cysouw -- I'm still not getting the latex to compile without error, specifically anywhere you use this type of construction (Undefined control sequence error):

\hlstd{}\hlslc{\#\ get\ the\ paths\ to\ the\ R\ executables\ in\ bash}\hspace*{\fill}\\

**Comment by bambooforest** [comment]
it knits fine, just fails to compile main.tex without being forced to in xelatex in TexShop using this installation.

http://www.tug.org/mactex/

i'll look into it, but also look at page 107 where one of the errors is occurring:

the alignment of the code is off:

# get the paths to the R executables in bash
   pathT=`Rscript -e 'cat(system.file("exec/tokenize", package="qlcData"))'`
   pathW=`Rscript -e 'cat(system.file("exec/writeprofile", package="qlcData"))'`

   # make softlinks to the R executables in /usr/local/bin
   # you will have to enter your user's password!
   sudo ln -is $pathT $pathW /usr/local/bin

could you please try to fix the alignment? not sure if the two issues are intertwined.

**Comment by cysouw** [comment]
fixed the alignment.

I now see the errors in latex as well: however: the end result is exactly what I expected, so I would simply ignore the errors :-). It is probably related to how knitr deals with bash-code.

**Comment by bambooforest** [comment]
@cysouw -- just a clarification please. where this fails to compile, these commands are meant for the user to run them (once) on the command line, or? e.g. 

```
# get the paths to the R executables in bash
   pathT=`Rscript -e 'cat(system.file("exec/tokenize", package="qlcData"))'`
   pathW=`Rscript -e 'cat(system.file("exec/writeprofile", package="qlcData"))'`
```
in that sense isn't a bit weird to have a comment before them? that would suggest, at least to me, that these codes chunks are meant in a bash script, instead of commands that should be fired off from the terminal. if that's the case, we can just incorporate the command line comments into the text -- and have no problems with knitr and latex :)  (sorry -- i really want this thing to compile without error.)

**Comment by cysouw** [comment]
OK, try it again.

**Comment by bambooforest** [comment]
Done. Thanks!

**Pull request merged**